intel_extension_for_transformers/backends/neural_engine/graph/models/model_utils/model_files.h

```diff
@@ -215,6 +215,8 @@ struct model_file_loader {
   }
   void read_vocab() {
     vocab.id_to_token.resize(hparams.n_vocab);
+    file.read_raw(&vocab.bos_token_id, sizeof(int));
```

Comment:
how about change `sizeof(int)` to `sizeof(model_vocab::id)`?

Reply:
good idea